### DISCUSSION TITLE
Event from LaravelEcho does not call given components method

### DISCUSSION THREAD

**Discussion by lostika86** [livewire/livewire#7471] [1. Help] [answered]
Hi, I need some help, I am broadcasting some event via pusher, (server side, client side works perfectly)
I made component, listeners are defined

```
public function getListeners()
    {
        return [
            "echo-private:cut-plan-movements,cut-plan-inserted" => 'notifyAlert',
        ];
    }


    #[On('echo-private:cut-plan-movements,cut-plan-inserted')]
    public function notifyAlert()
    {
         // does not run
         dd($this);

    }
```
it looks like the notifyAlert functions is not called
the network says data is coming

![image](https://github.com/livewire/livewire/assets/10046659/effb335d-440b-454f-9fcb-5e14113272f5)

**Comment by edalzell** [comment]
If I had to guess you missed the `.` that you need when it's a custom model event: https://laravel.com/docs/10.x/broadcasting#listening-for-model-broadcasts

![CleanShot 2023-12-06 at 11 06 08@2x](https://github.com/livewire/livewire/assets/6069653/01f9ad90-a1aa-4d31-86b9-cc0f91da7287)

**Reply by lostika86** [reply]
Thank you @edalzell, you have right, but if I have right `echo-private::` does this trick. If you carefully checks WS in network log you can see `private-` prefix is added before the event handle. 
https://livewire.laravel.com/docs/events#private--presence-channels
Any other idea?

**Reply by edalzell** [reply, marked as answer]
yes, you likely need this `echo-private:.cut-plan-movements` (note the `.`)

**Reply by lostika86** [reply]
Thank you for the notation, you have right.
The correct syntax is: `echo-private:cut-plan-movements,.cut-plan-inserted`
